SPECS/qemu-dp.spec

```diff
@@ -98,10 +98,16 @@ Patch87: make-blockdev-snapshot-use-same-node-name.patch
 Patch88: use_existing_io_context.patch
 Patch89: use_libaio_by_default.patch
 Patch90: Use_the_legacy_grant_copy_ioctl
+
+# XCP-ng patches
+Patch1000: qemu-dp-2.10.2-add-rbd-support.XCP-ng.patch
+
 BuildRequires: libaio-devel glib2-devel
 BuildRequires: libjpeg-devel libpng-devel pixman-devel libdrm-devel
 BuildRequires: xen-dom0-devel xen-libs-devel libusbx-devel
 BuildRequires: libseccomp-devel
+BuildRequires: librados-devel
+BuildRequires: librbd-devel
```

Comment:
I successfully built the package using librbd1-devel from CentOS. Why is that package not good enough? Is it too old?

Reply:
librbd1-devel from CentOS is pretty old. I'm not sure that it will work correctly with a ceph cluster of a newer version.  But it is possible to check. I will rebuild qemu-dp with librbd1-devel from CentOS and check it

Reply:
I checked it and it works correctly. So I think we can build qemu-dp with librbd1-devel from CentOS